src/ape/__modules__.py

```diff
@@ -1,15 +1,12 @@
+import re
+from pathlib import Path
```

Comment:
so we can use `find_packages` and parse out the plugins here but that would require adding `setuptools` as a dependency